portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,41 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Susana's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Welcome to My Portfolio:)!</h1>
+      <p>My name is Susana Perez<p> 
+      <h2>About me</h2>
+      <img src="images/download.png" alt="Front facing picture of a woman" <div id="size"></div>
```

Comment:
Not quite. This isn't valid HTML.
You never closed <img
You can't nest tags inside of each other (no <img ... <div> >, but <img> ... <div>... </div> ... </img> is fine)
Also, IDs should be unique on the page. So if you use "size" here you need to pick a different ID for the ones below. If you want to repeat, you can use a class (multiple <img class="size" ... /> is fine).

What I meant was something like <img id="download" ... />
Then in your css you can do
#download {
  ...
}

If you use classes <img class="myImg" ... /> then your css will be
.myImg {
 ...
}

Reply:
There are online validators that will check your HTML for you https://www.freeformatter.com/html-validator.html